tests/service/data/SyntaxTree/Expression/ObjectExpr 01.fs

```diff
@@ -0,0 +1,10 @@
+module V
+
+let create () =
```

Comment:
@nojaf Could you try to minimise the test data, please, so the things tested are easier to see?

Does having a binding cover something specific?
Does the second interface add something?

It seems the baseline would be much easier to reason about at a glance if there were less additional things. 🙂

Reply:
Yes, the first interface is currently spanning until the start of the second interface.
While the second interface ends at the `}` token.

The binding has no influence, can't say I care enough to update this sample.
Feel free to send me a PR 😉